### PR TITLE
Springster favicon

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
Please review 
skitch

File: gem/templates/springster/base.html [hunk 1]
```diff
@@ -30,6 +30,16 @@
     <meta property="og:image:width" content="450" />
     <meta property="og:image:height" content="200" />
     <meta name="viewport" content="width=device-width, initial-scale=1, {% if 'bbm.' in request.get_host %}user-scalable=0{% endif %}"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (nathanbegbie) [inline gem/templates/springster/base.html]: @Mitso does this not just repeat the line above it? - the BBM stuff will only get added if it's via the BBM viewport

**Comment by nathanbegbie** [comment]
@Mitso 1 comment, otherwise +1 from me

**Comment by Mitso** [comment]
That is indeed the case. Thanks @nathanbegbie

**Pull request merged**